docs/variants/workstation/installation-manual.md

```diff
@@ -0,0 +1,218 @@
+# Dasharo Workstation installation manual
+
+## Introduction
+
+__*Dasharo Workstation is a firmware product designed for Dell OptiPlex
+7010 and 9010*__
+
+In order to install the Dasharo Workstation you will need the Dell OptiPlex
+7010 or 9010 machine and a Linux installed on it (**in UEFI mode**
+since Dasharo Workstation v0.2 supports UEFI mode booting only).
+
+TBD: picture of whole workstation hardware, as it would be shown in shop
+
+## Preparation
+
+In order to install Dasharo Workstation one has to conduct a few preparation
+steps. Follow the below instructions:
+
+1. Open the case by lifting the handle on the case up.
+
+    ![](../../images/case_open1.jpg)
+
+2. Lift whole top cover and take it off.
+
+    ![](../../images/case_open2.jpg)
+
+3. Now it is time to release the disk dock. Lift up the handle of CD/DVD drive
+   bay.
+
+    ![](../../images/disk_dock_open1.jpg)
+
+4. Pull the CD/DVD drive bay to the CPU fan side.
+
+    ![](../../images/disk_dock_open2.jpg)
+
+5. Move the blue disk dock handle to the CPU fan side.
+
+    ![](../../images/disk_dock_open3.jpg)
+
+6. The screw should be at the bigger whole now. Lift up the whole dock to
+   remove it.
+
+    ![](../../images/disk_dock_open4.jpg)
+
+7. When the dock is removed the service mode jumper should be visible.
+
+    ![](../../images/overview_service_jumper.jpg)
+
+8. Place the jumper in the place marked by red rectangle.
+
+    ![](../../images/service_jumper.jpg)
+
+9. It should look like this.
+
+    ![](../../images/image0.jpeg)
+
+10. Power on the machine. You should see a warning that service jumper is
+active. Press F1 to proceed  and boot to your Linux system.
+
+    ![](../../images/service_mode_warn.jpg)
+
+## Installation
+
+You will need root privileges from now on to proceed. So switch to root user or
+use __*sudo*__ in each command.
+
+1. Install flashrom v1.1 with your distribution's package manager, if you don't
```

Comment:
v1.1 - why not v1.2 ? Also: if the success of installation could depend on a flashrom's version number, please provide the minimum flashrom version requirement. And point to the instruction like https://www.flashrom.org/Downloads in case the customer's flashrom version number provided by his Linux distro is insufficient: i.e. some ancient Debian could have a flashrom 0.x at repositories.

Reply:
@mikebdp2 you are right, please note in future fixes you can incorporate yourself. In most situations like this, I completely agree with your opinion. We will waste fewer resources on going back and fiorth.